### PR TITLE
Assert for forgetting .get() on an Ember proxy

### PR DESCRIPTION
**Pull request by chancancode**

### CODE REVIEW

File: packages/ember-metal/lib/property_get.js [hunk 1]
```diff
@@ -13,6 +14,23 @@ const ALLOWABLE_TYPES = {
...
+export const PROXY_CONTENT = symbol('PROXY_CONTENT');
+
+export function getPossibleMandatoryProxyValue(obj, keyName) {
+  if (MANDATORY_GETTER && EMBER_METAL_ES5_GETTERS && HAS_NATIVE_PROXY) {
+    let content = obj[PROXY_CONTENT];
+    if (content === undefined) {
+      return obj[keyName];
+    } else {
+      /* global Reflect */
+      return Reflect.get(content, keyName, obj);
```
Comment (rwjblue) [inline packages/ember-metal/lib/property_get.js]: Do we know of platforms that have native `Proxy` but don't support `Reflect`?

Comment (chancancode) [reply packages/ember-metal/lib/property_get.js]: Doesn't look like it: http://kangax.github.io/compat-table/es6/#test-Reflect (the opposite could be true though)

**Comment by rwjblue** [comment]
@chancancode - Where are we at with this?

**Comment by chancancode** [comment]
Probably need a cleanup and polish pass. Cleaning up the code/message/test and stuff. @wycats said he may be able to finish it this weekend.

**Review by rwjblue — APPROVED** [review]

**Pull request merged**